server/routes/course-route.js

```diff
@@ -12,6 +12,8 @@ const router = express.Router();
 router.get("/", courseRoutes.getCourses);
 router.get("/:courseid/lectures", lectureRoutes.getScheduledLectures);
 router.get("/:courseid/bookings", usageRoutes.getBookingStats);
+router.get("/:courseid/stats", usageRoutes.getCourseTotalStats);
```

Comment:
I think this route are not very well suited for the "course" route, they would make more sense under the "stats" route (e.g. `(stats)/:courseid`).